src/App.tsx

```diff
@@ -15,71 +15,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [chosenElement, chosenGood] = useState(goods[8]);
+  const [reset, onReset] = useState(true);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const clear = () => {
+    chosenGood(goods[-1]);
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+    onReset(prevState => !prevState);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+  return (
+    <main className="section container">
 
-        <tr data-cy="Good">
-          <td>
+      <h1 className="title is-flex is-align-items-center">
+        { chosenElement === goods[-1]
+          ? 'No goods selected'
+          : `${chosenElement} is selected` }
+        {reset
+          ? (
```

Comment:
Actually, this check is redundant. Just add it with the title if good is selected